game-core/src/main/java/games/strategy/engine/framework/startup/ui/pbem/PbemSetupPanel.java

```diff
@@ -214,6 +246,12 @@ public void update(final Observable o, final Object arg) {
   /** Called when the user hits play. */
   @Override
   public Optional<ILauncher> getLauncher() {
+    Preconditions.checkNotNull(
+        gameSelectorModel.getGameData(),
```

Comment:
The null assumption for `getGameData()` is a bit lax, this is an example of 'fail-fast' principle to ensure we have a good error message and secondly to document an assumption that the selector model game data should be set on game launch.